docs/components/slider/index.md

```diff
@@ -0,0 +1,27 @@
+<script setup>
+  import React from './react.md';
+  import Vue from './vue.md';
+  import Usage from './usage.md';
+</script>
+
+# Slider
+
+A slider is an input where the user selects a value from within a given range. The precise value, however, is not necessarily considered important. It can have single or dual drag handles.
```

Comment:
Do we have examples with dual drag handles?

Reply:
copy paste miss, not yet supported. I will remove the sentence